ocs_ci/ocs/resources/cbt_metadata.py

```
@@ -0,0 +1,869 @@
+        Parse the lister table output into structured data.
+
+        Args:
+            output (str): Raw table output from the lister tool
+
+        Returns:
+            list: List of dicts with keys Record, VolCapBytes,
+                BlockMetadataType, ByteOffset, SizeBytes
+        """
+        entries = []
+        for line in str(output).strip().split("\n"):
+            line = line.strip()
```

Ruff (0.16.1)

[warning] `for` loop variable `line` overwritten by assignment target

(PLW2901)